src/bencher/benchmark-db.sh

```diff
@@ -25,13 +25,13 @@ run_bench() {
   local log_file="$3"
 
   local bench_cmd="cargo run -r --bin bencher --features=bencher -- \
-    --path /slatedb-bencher_${put_percentage}_${concurrency} db \
+    --path /slatedb-bencher_${put_percentage}_${concurrency} --clean db \
```

Comment:
any chance we can parameterize the `benchmark-db.sh` script on this? I've actually found this super helpful to examine the results of the bencher after it runs as a learning exercise